README.md

```diff
@@ -212,7 +212,7 @@ This will be included when app is done.
 - [x] Controllers specs
 - [x] Creating a data model
 - [x] Processing data in models
-- [ ] Views
+- [x] Views
```

Comment:
- [ ] **[OPTIONAL]** I initially had issues running your app due to bootstrap. I was able to resolve it by installing bootstrap, jquery, and popper.js using the following command. It would be nice to add this, or any other preferred way to solve it to your documentation as well.

  ```
  yarn add bootstrap jquery popper.js
  ```
  ![fobara-bootstrap](https://user-images.githubusercontent.com/36542986/168067385-7d4b44b3-fdf1-49ea-8a22-ad9b1c999f98.jpg)

Reply:
Thank you. I will make sure I add it as a dependency with instructions on installation